.github/workflows/real-requests.yml

```diff
@@ -1,86 +1,53 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
+  workflow_dispatch:
```

Comment:
For this one I took R-CMD-check.yml with three edits

- adding the workflow dispatch to be able to manually trigger a run.
- adding the schedule stuff that was in the former version.
- adding the two VCR environment variables that were in the former version.